spec/acceptance/reboot_pending_spec.rb

```diff
@@ -24,14 +24,14 @@
       }
       MANIFEST
     windows_agents.each do |agent|
-      apply_manifest_on agent, undo_pending_reboot_manifest
+      execute_manifest_on(agent, undo_pending_reboot_manifest)
```

Comment:
Note that this is in `after(:all)` section and only applies to agents.  This is safe, and probably preferred, to a master-agent workflow